Week3/homework/step2-1.js

```diff
@@ -1,9 +1,7 @@
 'use strict';
 
 function foo(func) {
-  // What to do here?
-  // Replace this comment and the next line with your code
-  console.log(func);
+  return func();
```

Comment:
Sorry, I still don't understand how should I make the branches. Currently it is written:
	

> haha1906 wants to merge 3 commits into `HackYourFuture:master` from `haha1906:JS2-Week-3-Homework`

So I think that I've made `JS2-Week-3-Homework` branch. And my source branch was `master` branch of Hack your future. Should I have taken Week-2 branch, instead of master? How should my commit on top must be seen?